.github/pull_request_template.md

```diff
@@ -0,0 +1,15 @@
+\<Remember to add meaningful title\>
+
+\<Short description of the PR\>
+
+Fixes #\<issue number\>
+
+Before merge following needs to be applied:
+- [ ] At least one example testcase added in testcases/
+- [ ] Library Documentation regenerated according to [Generate docs](https://github.com/devopsspiral/KubeLibrary#development)
+- [ ] PR entry added in CHANGELOG.md in **In progress** section
+- [ ] All new testcases tagged as **prerelease** along other tags to exclude it from execution until released on PyPI
+- [ ] Coverage threshold increased in [.coveragerc](https://github.com/devopsspiral/KubeLibrary/blob/master/.coveragerc) if new coverage is higher than actual, see the lint-and-coverage step in CI
```

Comment:
I guess this includes adding unit tests for new keywords

Reply:
As discussed unit tests are encouraged, but at minimum it should not decrease coverage. Other than that coverage should gradually be up to some reasonable level.